From d032afcfa064634e0287c1d97b022d9eac16dc9f Mon Sep 17 00:00:00 2001
From: Michael Goffioul <michael.goffioul@lincor.com>
Date: Thu, 17 Oct 2019 15:13:50 -0400
Subject: [PATCH] Allow to use the network during build

This is required by external/kernel-drivers/wl/Android.mk
---
 BoardConfig.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index 8a5f34d..684a70d 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -94,4 +94,5 @@ BOARD_SEPOLICY_DIRS += device/generic/common/sepolicy \
 
 BOARD_BUILD_SYSTEM_ROOT_IMAGE:= true
 BUILD_BROKEN_PHONY_TARGETS := true
+BUILD_BROKEN_USES_NETWORK := true
 USE_XML_AUDIO_POLICY_CONF := 1
-- 
2.21.0

